p4-samples/vrf-routing/vrf.p4

```diff
@@ -0,0 +1,224 @@
+// Copyright 2020 Google LLC
```

Comment:
This file has really bizarre formatting. If you have time today, can you make it look like our P4 programs? e.g. no huge banners, different capitalization, indentation, line wrapping, etc

Reply:
done